### PR TITLE
[development] build new vernemq docker image

### PR DESCRIPTION
**Pull request by jonaphael**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Changed how we build the VerneMQ docker image

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Is there any issue related to this PR? (Link to an issue, e.g. #99999)** 
#1880 
* **Other information**:

### CODE REVIEW

File: connector/mqtt/vernemq/broker/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,22 @@
-FROM erlang:19.3.6.13 as basis
+# vernemq build
+FROM erlang:21-alpine AS build-env
```
Comment (mprevide) [inline connector/mqtt/vernemq/broker/Dockerfile:2]: erlang:21.3-alpine?
I think It's better fix the minor

File: connector/mqtt/vernemq/broker/bin/vernemq.sh [hunk 2]
```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env bash
+
```
Comment (mprevide) [inline connector/mqtt/vernemq/broker/bin/vernemq.sh:2]: Did you copy this file from some place?

Comment (mprevide) [reply connector/mqtt/vernemq/broker/bin/vernemq.sh:2]: I asked because of license, but maybe It shouldn't be a problem

Comment (jonaphael) [reply connector/mqtt/vernemq/broker/bin/vernemq.sh:2]: Yes I did
I just copy this repo https://github.com/sportebois/docker-vernemq/

Comment (mprevide) [reply connector/mqtt/vernemq/broker/bin/vernemq.sh:2]: The repo license is apache 2,0, what is good.
I think you just need refer the repo in the file
What do you think @rascaraficci ?

Comment (rascaraficci) [reply connector/mqtt/vernemq/broker/bin/vernemq.sh:2]: As this file was licensed under Apache 2.0 by the author, I think there is no problem to copy it. My suggestion is to add a NOTICE file saying that we copy/use it and the original one is available at ...

**Review by rascaraficci — COMMENTED** [review]
Good job. I left a minor comment.

**Review by mprevide — DISMISSED** [review]

File: connector/mqtt/vernemq/broker/bin/LICENSE_NOTICE [hunk 3]
```diff
@@ -0,0 +1,10 @@
+##################################################################
```
Comment (rascaraficci) [inline connector/mqtt/vernemq/broker/bin/LICENSE_NOTICE:1]: I have some suggestions:

- Change the filename just to NOTICE

- Change the file's content by something like:

To meet the needs of dojot IoT Platform, it is built a customized VerneMQ image. The Dockerfile and scripts used to build this image were based on code from https://github.com/sportebois/docker-vernemq, which are licensed under the Apache 2.0 license.

**Review by rascaraficci — CHANGES_REQUESTED** [review]
I left a minor comment.

**Review by mprevide — APPROVED** [review]

**Review by rascaraficci — APPROVED** [review]
It looks OK now!

**Pull request merged**